src/content/plugins/progress-plugin.md

```diff
@@ -6,11 +6,22 @@ contributors:
   - byzyk
 ---
 
+`object` `function (percentage: number, message: string, ...args: string[])`
+
 The `ProgressPlugin` provides a way to customize how progress is reported during a compilation.
 
 ## Usage
 
-Create an instance of `ProgressPlugin` with a handler function which will be called when hooks report progress:
+Create an instance of `ProgressPlugin` and provide one of the allowed params.
+
+### Providing `function`
+ 
+Provide a handler function which will be called when hooks report progress. `handler` function arguments:
+
+- `percentage`: a number between 0 and 1 indicating the completion percentage of the compilation
+- `message`: a short description of the currently-executing hook
+- `...args`: zero or more additional strings describing the current progress
```

Comment:
> zero or more additional strings

That doesn't say anything haha, What is it exactly?

Reply:
😅 
Yes unfortunately source code doesn't have them used. I think it's reserved to avoid breaking changes in future when plugin is changed